common/dao-api/src/main/java/org/thingsboard/server/dao/user/UserService.java

```diff
@@ -60,6 +60,8 @@ public interface UserService extends EntityDaoService {
 
     PageData<User> findTenantAdmins(TenantId tenantId, PageLink pageLink);
 
+    PageData<User> findUsers(TenantId tenantId, PageLink pageLink);
```

Comment:
Please rename all similar methods to ```findAllUsers```. I suspect someone may use this method and expect to find users by tenant id. But this method returns all users disregarding the tenant id.

Reply:
TenantId is used only to count db calls and some other stats